extension/src/popup/components/WarningMessages/index.tsx

```diff
@@ -111,21 +118,29 @@ const DangerousAccountWarning = ({
   return null;
 };
 
-const MemoWarningMessage = ({ isMemoRequired }: { isMemoRequired: boolean }) =>
-  isMemoRequired ? (
+const MemoWarningMessage = ({
+  isMemoRequired,
+}: {
+  isMemoRequired: boolean;
+}) => {
+  const { t } = useTranslation();
+
+  return isMemoRequired ? (
     <WarningMessage header="Memo is required" isHighAlert>
       <p>
-        A destination account requires the use of the memo field which is not
-        present in the transaction you’re about to sign. Freighter automatically
-        disabled the option to sign this transaction.
+        {t(
+          "A destination account requires the use of the memo field which is not present in the transaction you’re about to sign. Freighter automatically disabled the option to sign this transaction.",
```

Comment:
I'm not *entirely* sure what the best approach for long blocks of text is yet. You'll see that translation builder creates a deeply nested object out of paragraphs because of the periods present in the text. It's a little unsightly in the JSON, but keeps the meaning in the code

I'm open to changing this if this becomes a mess